dbt/tests/data_tests/generic_tests/expect_fgd_cost_totals_to_agree.sql

```diff
@@ -0,0 +1,26 @@
+{% test expect_fgd_cost_totals_to_agree(model) %}
+with Discrepancies as (
+    select
+        case
+            when (fgd_structure_cost is not null) or (fgd_other_cost is not null) or (sludge_disposal_cost is not null) then coalesce(fgd_structure_cost, 0) + coalesce(fgd_other_cost, 0) + coalesce(sludge_disposal_cost, 0)
```

Comment:
I could see this type of test being helpful in other contexts. Something like "make sure sums = total". Is it worth adding parameters instead of hardcoding the columns? 

That may not be feasible given the treatment of certain null values here.

Reply:
This one can't be refactored as a percentage_failure because of the weird null handling, but a more-generic one in general sounds like a good idea. istr we're concerned about budget on the migration in general though so is this a blocker for you or a nice-to-have? aka should I do this refactor now, or wait to see if we have hours left once everything else has been migrated?

Reply:
It's more of a nice to have I'd say!